### PR TITLE
Feature components

### PR DESCRIPTION
**Pull request by carloso0114**
Create a React component that matches the calculator design, and import it on App.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <p> Hello world </p>
-      </header>
-    </div>
+    <Calculator />
   );
```
Comment (DeliceLydia) [inline src/App.js]: - [x] The `App.js` component should be class-based please work on that

File: src/index.js [hunk 2]
```diff
@@ -11,7 +11,4 @@ ReactDOM.render(
   document.getElementById('root'),
 );
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
 reportWebVitals();
```
Comment (DeliceLydia) [inline src/index.js]: - [x] Delete all the boilerplate that was created that you will not need in your app

File: src/setupTests.js [hunk 3]
```diff
@@ -1,5 +1 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
```
Comment (DeliceLydia) [inline src/setupTests.js]: - [x] All of these are not needed in your app kindly delete them

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @carloso0114👏,

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @carloso0114,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**